CHANGELOG.md

```diff
@@ -7,37 +7,33 @@
 ### Added
 
 * Support Ruby 3.2 ([#2601][])
-* Create init image for lib-injection ([#2606][])
+* Publish init container image (beta) for `dd-trace-rb` injection through K8s admission controller ([#2606][])
 * Tracing: Support 128 bits trace id  ([#2543][])
-* Tracing: Add tags to  integrations (`que` / `racecar` / `resque`/ `shoryken` / `sneakers` / `qless` / `delayed_job` / `kafka` / `sidekiq` / `dalli` / `presto` / `elasticsearch`) ([#2619][],  [#2613][] , [#2608][], [#2590][])
+* Tracing: Add tags to integrations (`que` / `racecar` / `resque`/ `shoryken` / `sneakers` / `qless` / `delayed_job` / `kafka` / `sidekiq` / `dalli` / `presto` / `elasticsearch`) ([#2619][],  [#2613][] , [#2608][], [#2590][])
 * Appsec: Introduce `AppSec::Instrumentation::Gateway::Argument` ([#2648][])
```

Comment:
That is also internal. Similar to the appsec component. So we might be able to remove it as well